mteb/evaluation/MTEB.py

```diff
@@ -455,9 +459,19 @@ def run(
                 f"\n\n********************** Evaluating {task.metadata.name} **********************"
             )
 
+            task_eval_splits = (
+                eval_splits if eval_splits is not None else task.eval_splits
+            )
+            task_eval_langs = (
+                task.metadata.eval_langs
```

Comment:
`TaskMetadata.hf_subsets_to_langscripts` standardized everything to a dict I would use this throughout